lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -41,7 +41,14 @@ use virt_utils;
 
 our @EXPORT
   = qw(download_network_cfg prepare_network restore_standalone destroy_standalone restart_network restore_guests restore_network
-  destroy_vir_network restore_libvirt_default enable_libvirt_log upload_debug_log check_guest_status check_guest_module save_guest_ip test_network_interface hosts_backup hosts_restore);
+  destroy_vir_network restore_libvirt_default enable_libvirt_log upload_debug_log check_guest_status check_guest_module save_guest_ip test_network_interface hosts_backup hosts_restore get_available_pool);
+
+sub get_available_pool {
+    if (is_xen_host()) {
+        my $pool =  script_output q@virsh pool-info default | grep ^Available | awk '{print $2}'@;
```

Comment:
I would add
```perl
script_run 'df -h';
```
just to have some debug output.

Reply:
Great, agree on this. Had added some debug output about hard disk info

Reply:
DONE